fixes code and screenshot inconsistencies

noodep opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description
~~Fixes names used in code snippets to match names shown in screenshots of code output.~~
Replaces screenshots with outdated names with markdown table containing names matching associated code snippets
<!-- ✍️ Summarize your changes in one or two sentences -->

### Motivation
This inconsistency makes it difficult for readers to link the behavior of the code to its actual shown output.
<!-- ❓ Why are you making these changes and how do they help readers? -->

### Additional details
N/A
<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests
None
<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->


<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

wbamberg reviewed (commented):
Thanks for your PR, @noodep !

These names were changed in https://github.com/mdn/content/pull/21831. That PR didn't update the images, but we still want the new names.

Actually, I think it would be best to replace these images with just normal tables. We should be able to user Markdown tables for them (https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#gfm_table_syntax_style) as they are not big or complicated.

noodep committed:
fix: fixes code and screenshot inconsistencies

replaces screenshots of code output containing outdated names with markdown tables containing names matching the ones used in associated code snippets
removes now unused images

noodep commented:
Ah I see, thank you for pointing this out @wbamberg.

I replaced the screenshots with markdown tables as described in the link you sent and I agree that these should be sufficient.

wbamberg reviewed (approved):
👍 looks great @noodep , thank you for the fix!
